src/main.cpp

```diff
@@ -87,6 +87,9 @@ arith_uint256 nBestChainTrust = 0;
 arith_uint256 nBestInvalidTrust = 0;
 uint256 hashBestChain;
 CBlockIndex* pindexBest = NULL;
+// Lets try to start using some lockless synchronization.
+std::atomic<int64_t> nSyncTime {0};
+std::atomic_bool fOutOfSyncByAge {true};
```

Comment:
Think we should use the `g_` prefix for these? We've been following that convention to denote global variables for new code.

Reply:
Yeah... I agree, although I will also leave the small f in there... so g_fOutOfSyncByAge?